frontend/test/metabase/scenarios/models/reproductions/22715-remapped-values-override-column-identifier.cy.spec.js

```diff
@@ -26,10 +26,10 @@ describe.skip("filtering based on the remapped column name should result in a co
 
       // Let's go straight to the model metadata editor
       cy.visit(`/model/${id}/metadata`);
-      // Without this Cypress fails to remap the column because an element becomes detached from the DOM
-      cy.findByText(
-        "Use the tab key to navigate through settings and columns.",
-      );
+      // Without this Cypress fails to remap the column because an element becomes detached from the DOM.
+      // This is caused by the DatasetFieldMetadataSidebar component rerendering mulitple times.
+      cy.findByText("Database column this maps to");
+      cy.wait(5000);
```

Comment:
This is awful, but the component this tests rerenders like 10 times, so you run into situations where the element this test finds has been detached from the DOM. Waiting for 5 secs lets the component calm down.

https://user-images.githubusercontent.com/13057258/184419208-fbe2675e-5480-42ca-ad97-dd2817c0e96d.mov

Reply:
Noted, thanks.